### DISCUSSION TITLE
Rclone mount fails on RH7.9 aarch64

### DISCUSSION THREAD

**Discussion by eugfir** [libfuse/libfuse#860] [Q&A]
Hello, 

I loaded fuse3 on my RedHat 7.9 aarch64 linux from https://archlinuxarm.org/packages/aarch64/fuse3 which seems to be the latest build.  When I am trying to mount onedrive filesystem configured by rclone I am getting the following error:

[root@inst1 ~]# rclone --vfs-cache-mode writes mount onedrive: /onedrive
2023/11/13 16:39:27 mount helper error: fusermount3: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by fusermount3)
2023/11/13 16:39:27 mount helper error: fusermount3: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by fusermount3)
2023/11/13 16:39:27 Fatal error: failed to mount FUSE fs: fusermount: exit status 1

To me the problem being that I have GLIBC_2.17 on RH7.9 instead. Is there a way to get a fuse3 older build compatible with glibc-2.17.aarch64 ?

Thanks.
-Eugene

**Comment by Nikratio** [comment]
You should be able to just build it from source yourself, yes.

**Comment by eugfir** [comment]
Hi Nikratio,

I tried to follow your suggestions but gave up - too many moving parts to setup to make one single build (Meson, Ninja, etc,,,BUMMER! - why not just simply good old Maven or Gradle??). Ended up with upgrading RHEL7.9 Linux server to RHEL8, still got GLIBC lower than required (2.28) but that did the trick so was able to mount OneDrive from rclone. All good now!

Thanks,
-Eugene

**Reply by bsbernd** [reply]
We actually have a build environment with centos7, using meson and ninja. I also don't see where the glibc requirement should come from. 
I don't know what "Maven" or "Gradle" are, in the past the project was using autotools and personally I very much prefer meson (speed, syntax (m4...), out-of-source builds, etc).

Anyway, seems to work for you now, so closing the discussion.
